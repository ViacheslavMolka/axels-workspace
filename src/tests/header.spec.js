import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../components/header';


describe('Header component', () => {
    it('Header snapshot', () => {
        const component = renderer.create(<Header/>).toJSON()
        expect(component).toMatchSnapshot()
    });
});
