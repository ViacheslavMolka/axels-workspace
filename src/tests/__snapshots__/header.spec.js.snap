// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header snapshot 1`] = `
<div
  className="sc-eCssSg gWZgXy header"
>
  <span
    className="navbar-brand"
  >
    <svg
      fill="#cf4dff"
      height={29}
      viewBox="0 0 16 16"
      width={29}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M10.478 1.647a.5.5 0 1 0-.956-.294l-4 13a.5.5 0 0 0 .956.294l4-13zM4.854 4.146a.5.5 0 0 1 0 .708L1.707 8l3.147 3.146a.5.5 0 0 1-.708.708l-3.5-3.5a.5.5 0 0 1 0-.708l3.5-3.5a.5.5 0 0 1 .708 0zm6.292 0a.5.5 0 0 0 0 .708L14.293 8l-3.147 3.146a.5.5 0 0 0 .708.708l3.5-3.5a.5.5 0 0 0 0-.708l-3.5-3.5a.5.5 0 0 0-.708 0z"
      />
    </svg>
     
    <span
      className="frontend-text"
    >
      Front-end Developer Test Task
    </span>
  </span>
  <span
    className="navbar-brand"
  >
    <span
      className="cart-text"
    >
      cart
    </span>
    <svg
      fill="#cf4dff"
      height={19}
      viewBox="0 0 16 16"
      width={19}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 1.5A.5.5 0 0 1 .5 1H2a.5.5 0 0 1 .485.379L2.89 3H14.5a.5.5 0 0 1 .49.598l-1 5a.5.5 0 0 1-.465.401l-9.397.472L4.415 11H13a.5.5 0 0 1 0 1H4a.5.5 0 0 1-.491-.408L2.01 3.607 1.61 2H.5a.5.5 0 0 1-.5-.5zM3.102 4l.84 4.479 9.144-.459L13.89 4H3.102zM5 12a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm7 0a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm-7 1a1 1 0 1 1 0 2 1 1 0 0 1 0-2zm7 0a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"
      />
    </svg>
    <span
      className="circle"
    />
  </span>
</div>
`;

exports[`Header snapshot 1`] = `ShallowWrapper {}`;
